dynawo/sources/Models/CPP/ModelNetwork/DYNModelNetwork.cpp

```diff
@@ -986,11 +986,10 @@ ModelNetwork::initParams() {
   getY0();
 
   // solve initial problem
-  SolverSubModel solver;
-  solver.init(this, 0, &yLocalInit_[0], &fLocalInit_[0]);
+  SolverKINSubModel solver;
+  solver.init(this, 0, &yLocalInit_[0], &fLocalInit_[0], 5);
```

Comment:
Clearer by using a constant (const nbMaxIter = 5;). Also, please put above or  remove the comments below

Reply:
Done